src/Microsoft.Diagnostics.Monitoring.Options/MonitorCapabilityConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Diagnostics.Monitoring.Options
+{
+    internal static class MonitorCapabilityConstants
+    {
+        public const string Exceptions = "exceptions";
+        public const string ParameterCapturing = "parameters";
+        public const string CallStacks = "callstacks";
```

Comment:
nit: Call stacks is two words; let's put an underscore between the two words:
```suggestion
        public const string CallStacks = "call_stacks";
```